src/js/components/ErrorInformation/index.jsx

```diff
@@ -7,33 +7,39 @@ import {Button} from 'reactstrap';
 import errorImage from '../../../images/error.png';
 import './index.scss';
 
-const ErrorInformation = ({t, provider, network, transaction, sntTokenError, retry, message, cancel}) => (
+const ErrorInformation = ({t, provider, network, transaction, sntTokenError, retry, message, cancel, customErrorTitle, customErrorTip, cancelText, CTAText, r}) => (
   <div className={classnames("error-information with-tip", {'with-button': !!retry})}>
     <img src={errorImage} alt="error"/>
     <h2 className="mt-5">
       {provider && t('errorInformation.provider.title')}
       {network && t('errorInformation.network.title')}
       {transaction && t('errorInformation.transaction.title')}
       {sntTokenError && t('errorInformation.sntTokenError.title')}
+      {customErrorTitle && t(customErrorTitle)}
     </h2>
     {message && <p className="text-muted mb-2">{message}</p>}
     <p className="text-muted mb-2">
       {provider && t('errorInformation.provider.tip')}
       {network && t('errorInformation.network.tip', {network})}
       {transaction && t('errorInformation.transaction.tip')}
       {sntTokenError && t('errorInformation.sntTokenError.tip')}
+      {customErrorTip && t(customErrorTip)}
     </p>
     <p>
-      {cancel && <Button color="secondary" className="mr-3" onClick={cancel}>{t('errorInformation.cancel')}</Button>}
-      {retry && <Button color="primary" onClick={retry}>{t('errorInformation.retry')}</Button>}
+      {cancel && <Button color="secondary" className="mr-3" onClick={cancel}>{t(cancelText)}</Button>}
+      {retry && <Button color="primary" onClick={retry}>{t(CTAText)}</Button>}
```

Comment:
Same for all the other. YOu can replace them by 
```suggestion
      {retry && <Button color="primary" onClick={retry}>{CTAText}</Button>}
```